docs/content/developers/building-contributing.md

```diff
@@ -26,70 +26,87 @@ Each [PR build](https://github.com/ddev/ddev/actions/workflows/pr-build.yml) cre
+        xattr -r -d com.apple.quarantine /usr/local/bin/ddev
+        ```
 
-```
-sudo rm /usr/local/bin/ddev
-brew link --force ddev
-```
+        (The binaries on the master branch and the final release binaries _are_ signed.)
 
-### Installing a Downloaded Binary in the `$PATH`
+    After you’re done testing, you can delete your downloaded executable, and re-link the original Homebrew one:
 
-Normally, you can put any executable in your path, and it takes precedence, so you don't need to remove or disable an already installed DDEV instance, which we will use here. This example uses `~/bin`. `echo $PATH` and `which ddev` are valuable commands for debugging. Since not every distro has `~/bin` in `$PATH`, you can create the folder and add it to your path in `~/.bashrc` with these commands:
+    ```bash
+    sudo rm /usr/local/bin/ddev
+    brew link --force ddev
+    ```
 
-```
-mkdir -p ~/bin
-export PATH="~/bin:$PATH"
-```
+=== "Installing a Downloaded Binary in the `$PATH`"
 
-Next, unzip the ZIP file you downloaded, make it executable, and move it to a folder in your path. Check with `echo $PATH`:
+    ### Installing a Downloaded Binary in the `$PATH`
 
-```
-unzip ddev.zip
-chmod +x ddev && mv ddev ~/bin
-```
+    Normally, you can put any executable in your path, and it takes precedence, so you don't need to remove or disable an already installed DDEV instance, which we will use here. This example uses `~/bin`. `echo $PATH` and `which ddev` are valuable commands for debugging. Since not every distro has `$HOME/bin` in `$PATH`, you can create the folder and add it to your path in `~/.bashrc` with these commands:
 
-Now, close and reopen your terminal, and verify the replacement worked by running `ddev version`. The output should be something like `DDEV version v1.22.3-39-gfbb878843`, instead of the regular `DDEV version v1.22.3`.
+    ```bash
+    mkdir ~/bin
+    echo 'export PATH="$HOME/bin:$PATH"' >>~/.bashrc
+    source ~/.bashrc
+    # Verify that `~/bin` was added to your $PATH
+    echo $PATH
+    ```
 
-You need to run `ddev poweroff` and `ddev start` to download the Docker images that it needs.
+    Next, download the ZIP archive, unzip it, make it executable, and move it to the `~/bin` folder:
 
-After you’re done testing, you can delete your downloaded executable, restart your terminal, and again use the standard DDEV:
+    ```bash
+    wget -O ddev.zip [LINK TO ZIP ARCHIVE]
+    unzip ddev.zip
+    chmod +x ddev && mv ddev ~/bin/ddev
+    rm ddev.zip
+    ```
 
-```
-rm ~/bin/ddev
-```
+    Replace `[LINK TO ZIP ARCHIVE]` with the corresponding link from the PR comment:
+
+    ![Github Action PR Comment](../images/github-action-pr-comment.png)
+
+    Verify the replacement worked by running `ddev -v`. The output should be something like `ddev version v1.23.5-98-g3c93ae87e`, instead of the regular `ddev version v1.23.5`.
+
+    You need to run `ddev poweroff` and `ddev start` to download the Docker images that it needs.
```

Comment:
```suggestion
    When DDEV detects a version change, it recommends powering down all running containters. Then, it will download the new images, if required.
```

DDEV does not need to "poweroff" to get new images:

```shell 
$ ddev start
You seem to have a new DDEV version.
During an upgrade it's important to `ddev poweroff`.
May I do `ddev poweroff` before continuing?
This does no harm and loses no data. [Y/n] (yes): n
Starting cypress...
20241122_stasadev_php_pecl: Pulling from ddev/ddev-webserver
132e290b53a3: Download complete
Digest: sha256:1111b1a5766693de8b432149941433a432c8d53f41c24ba1c1efd443a6e6f912
Status: Downloaded newer image for ddev/ddev-webserver:20241122_stasadev_php_pecl
```